packages/lib/slots.test.ts

```diff
@@ -210,7 +210,9 @@ describe("Tests the slot logic", () => {
       inviteeDate: dayjs().add(1, "day"),
       frequency: 20,
       minimumBookingNotice: 0,
-      dateRanges: [{ start: dayjs("2021-06-21T00:00:00.000Z"), end: dayjs("2021-06-21T23:45:00.000Z") }],
+      dateRanges: [
+        { start: dayjs("2021-06-21T00:00:00.000Z").utc(), end: dayjs("2021-06-21T23:45:00.000Z").utc() },
```

Comment:
@MehulZR In which format does the caller send these dates? Utc? I want to make sure that the test and the production code send the dates in the same format

Reply:
Dates can be in any tz as Dayjs Object.

Reply:
Yeah I know but I would like the tests to call the functions like the production code calls them so we expect and see consistent results.

Reply:
In production, the fn is called with different timezones. and the result (no. of slots) is dependent on the that timezone.
So, In test I have converted the tz into UTC because the result is correct for the UTC tz.